spec/requests/posts_spec.rb

```diff
@@ -2,6 +2,42 @@
 
 RSpec.describe 'Posts', type: :request do
   describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+    before(:each) do
+      @user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
+      # @post = Post.new(author: @user, title: 'Hello', text: 'This is my first post')
+      get '/users/1/posts'
+    end
+
+    it 'returns http success' do
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders correct template' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'response body includes correct placeholder text' do
+      expect(response.body).to include('Here is a list of posts for a given user')
+    end
+  end
+
+  describe 'GET /index' do
+    before(:each) do
+      @user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
+      @post = Post.new(author: @user, title: 'Hello', text: 'This is my first post')
+      get '/users/1/posts/1'
+    end
+
+    it 'returns http success' do
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders correct template' do
+      expect(response).to render_template(:show)
+    end
+
+    it 'response body includes correct placeholder text' do
+      expect(response.body).to include('Here is a list of posts given a user id and post id')
+    end
```

Comment:
- [ ] Great job working on the tests so far but I think these three tests are not passed on my side so please I highly recommend to work on them to let them pass like the other tests 👍🏻🚀✅🤩💯 I think because of line 27 when I tried to comment it out it works just fine so please comment it out to let the tests pass 🤩💯🔥🔥


![image](https://github.com/Rhaegar121/Blog-app/assets/58553711/5a9d04ec-6d1a-4eae-bfe3-5f769381b01e)